it/controllers/agent/InYearTaxCalculationControllerISpec.scala

```diff
@@ -119,7 +119,7 @@ class InYearTaxCalculationControllerISpec extends ComponentSpecBase {
   }
 
   object ExpectedValues {
-    val title = messages(s"htmlTitle.agent")
+    val title = messages(s"agent.header.serviceName") + " - " + messages("base.govUk")
```

Comment:
Same with this - this message key is for the black banner (service name in header)